src/app/home/home.page.html

```diff
@@ -48,9 +48,9 @@ <h1>{{banner?.title}}</h1>
         <ion-grid>
           <ion-row>
             @for (item of solutionList; track $index) {
-              @if (item.type === 'solutionList'){
-                @for (listData of item.listingData; track listData.id) {
-            <ion-col size="6" size-md="3" (click)="navigateTo(listData)">
+            @if (item.type === 'solutionList'){
+            @for (listData of item.listingData; track listData.id) {
+            <ion-col size="6" size-md="3" [routerLink]="listData?.redirectionUrl" routerDirection="forward">
```

Comment:
When we re-direct from home to listing without sending the data, wouldn't this affect the listing page where we are expecting the data to get from the url? what if we want to send nested data (which will come during survey lisitng)?